.github/workflows/maven.yml

```diff
@@ -36,6 +36,10 @@ jobs:
         server-password: MAVEN_PASSWORD
         gpg-private-key: ${{ secrets.GPG_SECRET_KEY }}
         gpg-passphrase: MAVEN_GPG_PASSPHRASE
+    - name: Extend GPG key expiration
+      run: |
+        echo -e "expire\n1y\nsave" > gpg_commands.txt
+        gpg --batch --yes --pinentry-mode loopback --passphrase "${{ secrets.GPG_PASSPHRASE }}" --command-file gpg_commands.txt --edit-key C2B7EEEC51372BAB
```

Comment:
Not sure where this edit-key value comes from? Were you able to list keys somewhere to get this or should this be coming from an env var instead?

Reply:
yea i just did a gpg list-keys on my hotfix branch. This key is imported so it should always have the same key id.